### PR TITLE
@kitfoman Add ability to connect wallet, expose contract methods to redux, move… …

### PR DESCRIPTION
**Pull request by tgetachew18**

### CODE REVIEW

**Review by btree1970 — APPROVED** [review]

File: contracts/pool.code-workspace [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+	"folders": [
+		{
+			"path": "/Users/aida/coding/hackmoney-poolio"
```
Comment (ksgebrekirstos98) [inline contracts/pool.code-workspace:4]: What is the purpose of this file? Also, let's avoid using absolute paths.

File: contracts/pool.code-workspace [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+		},
+		{
+			"path": "/Users/aida/coding/Banking-As-A-Service"
```
Comment (ksgebrekirstos98) [inline contracts/pool.code-workspace:7]: Typically **contracts** directory is reserved for .sol files

File: frontend/src/contracts/Token.json [hunk 3]
```diff
@@ -0,0 +1,105 @@
+{
```
Comment (ksgebrekirstos98) [inline frontend/src/contracts/Token.json:1]: Do we need this file? We're not using the Token.sol contract

File: frontend/src/features/home/home.css [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+}
+
+.sidepanel-bg {
+    background-color: white;
+    height: 100%;
+    padding: .5em;
+}
+
+header {
+    /* -webkit-box-shadow: 0px 0px 9px 3px rgba(41,41,41,.25);
```
Comment (ksgebrekirstos98) [inline frontend/src/features/home/home.css:26]: Remove commented lines

File: frontend/src/redux/poolSlice.js [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+// We'll use ethers to interact with the Ethereum network and our contract
+import { ethers } from "ethers";
+// We import the contract's artifacts and address here, as we are going to be
+// using them with ethers
+import PoolFactoryArtifact from '../contracts/PoolFactory.json';
+import PoolArtifact from '../contracts/Pool.json';
+import contractAddress from '../contracts/contract-address.json';
+import { createSlice } from '@reduxjs/toolkit';
+
+// // Define a thunk that dispatches those action creators
```
Comment (ksgebrekirstos98) [inline frontend/src/redux/poolSlice.js:12]: Remove unnecessary comments in this file

File: test/dappSlice.js [hunk 6]
```diff
@@ -0,0 +1,166 @@
+import React from "react";
+
+// We'll use ethers to interact with the Ethereum network and our contract
```
Comment (ksgebrekirstos98) [inline test/dappSlice.js:3]: Remove unnecessary comments in all places in this file

File: tasks/faucet.js [hunk 7]
```diff
@@ -25,17 +25,18 @@ task("faucet", "Sends ETH and tokens to an address")
...
       return;
     }
 
-    const token = await ethers.getContractAt("Token", address.Token);
+    const token = await ethers.getContractAt("PoolFactory", address.PoolFactory);
     const [sender] = await ethers.getSigners();
 
-    const tx = await token.transfer(receiver, 100);
-    await tx.wait();
+    // const tx = await token.transfer(receiver, 100);
```
Comment (ksgebrekirstos98) [inline tasks/faucet.js:36]: Remove comments

**Review by ksgebrekirstos98 — APPROVED** [review]

**Comment by ksgebrekirstos98** [comment]
Also, please choose the option "Squash and merge" when you commit your changes. We want master to have a linear history

File: contracts/pool.code-workspace (hunk 1 above)
Comment (btree1970) [reply contracts/pool.code-workspace:4]: lmao

File: frontend/src/features/home/home.css (hunk 4 above)
Comment (booleanings) [reply frontend/src/features/home/home.css:26]: lol it's not that deep homie. we not getting graded on the code

**Pull request closed without merging**